streamer/parser/metadata.md

```diff
@@ -291,48 +293,68 @@ Does not apply.
 
 The integer is the value of the `meta` element whose `property` attribute has the value `schema:numberOfPages`.
 
-## Rendition
+## Rendition / Presentation
 
-The `rendition` of a publication is an object containing the following keys: 
+The `presentation` of a publication is an object containing the following keys: 
```

Comment:
We should also list `layout`.

Reply:
Ah if you can have it in presentation (wasn't sure) then it also simplify the layout section.

Reply:
Yes you can: https://readium.org/webpub-manifest/schema/extensions/epub/metadata.schema.json

Reply:
Done